### ISSUE TITLE
NTR: Menstrual cycles have begun  (General history, localization & modifiers)

### ISSUE THREAD

**Issue by pnrobinson** [obophenotype/human-phenotype-ontology#7855]
New term request Menstrual cycles have begun  (brumme_220429194623): Menstruating female, menarche, reproductive period between menarche and menopause  UMLS: 0025274

**Comment by pnrobinson** [comment]
@MickeySegal

does this term intend to refer to the onset of a disease?

**Comment by MickeySegal** [comment]
No, it is a puberty stage finding, opening up the possibility of pathological pregnancies such as ectopic pregnancy, or premenstrual syndrome.

**Comment by pnrobinson** [comment]
@MickeySegal 
Adding new term: 
- ID: HP:6000309
- label: Menstruating female between menarche and menopause
- parents: [Past medical history (HP:0032443)](https://hpo.jax.org/app/browse/term/HP:0032443)
- pmid: 
- def: Past medical history of a woman who is menstruating, i.e., not premenstrual or post-menopause.
- comment: 
- orcid: orcid:0000-0002-0736-9199
- github ID: https://github.com/obophenotype/human-phenotype-ontology/issues/7855
- synonyms:
